### PR TITLE
Fix: Filter by teacher in admin

### PR DESCRIPTION
**Pull request by geordi**
Displays only teacher logins in the filter box. Fixes Issue #2

### CODE REVIEW

File: common/admin.py [hunk 1]
```diff
@@ -4,10 +4,40 @@
...
+    parameter_name = 'teacher'
+
+    def lookups(self, request, model_admin):
+        teachers = User.objects.filter(groups__name='teachers')
+        items = ( (t.id, t.username) for t in teachers )
+
+        return tuple(items)
+
+    def queryset(self, request, queryset):
+        pass
```
Comment (trnila) [inline common/admin.py]: it can be omitted

File: common/admin.py [hunk 2]
```diff
@@ -4,10 +4,40 @@
...
+    and https://stackoverflow.com/questions/880684/in-django-admin-how-to-filter-users-by-group
+    """
+    title = 'teacher'
+    parameter_name = 'teacher'
+
+    def lookups(self, request, model_admin):
+        teachers = User.objects.filter(groups__name='teachers')
+        items = ( (t.id, t.username) for t in teachers )
+
+        return tuple(items)
```
Comment (trnila) [inline common/admin.py]: `items` is already a tuple, so it can be returned immediately

Comment (geordi) [reply common/admin.py]: Well, `items` is a generator and I didn't know it'd work. I can definitely test it.

File: common/admin.py (hunk 1 above)
Comment (geordi) [reply common/admin.py]: Wanted to be explicit.

**Pull request merged**